Dockerfile

```diff
@@ -1,13 +1,24 @@
-FROM node:10
+FROM node:10 AS bootstrap
 
 RUN mkdir -p /usr/src
 WORKDIR /usr/src/
 
-COPY ./ .
+COPY ./ /usr/src
 RUN chown -R node:node .
 
 USER node
 
 RUN yarn install
 
 RUN yarn bootstrap
+
+FROM node:10 AS production-apps
+
+RUN mkdir -p /usr/src
+WORKDIR /usr/src/
+
+COPY --from=bootstrap /usr/src /usr/src
```

Comment:
Ideally, this step should only copy over the files that are needed to run the production apps, not the entire source tree, tests etc.